manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -632,6 +632,8 @@ webhooks:
         namespace: istio-system
         path: "/inject"      
       caBundle: 
+    sideEffects: None
```

Comment:
Can you comment on why the sideEffects is none for MutatingWebhookConfiguration?

Reply:
It's a required field in apiVersion `admissionregistration/v1`